controlSystem/RecoveryBoard/firmware-rs/Cargo.toml

```diff
@@ -21,6 +21,17 @@ embedded-hal-nb = "1.0.0"
 noline = "0.5.1"
 heapless = "0.9.1"
 format_no_std = "1.2.0"
+embedded-test-linker-script = { version = "0.1.0" }
+
+[dev-dependencies]
+embedded-test = { version = "0.7.0", features=["embassy", "defmt"] }
```

Comment:
Non-blocking:  how does a developer choose the "features" for each item in the sections "dependency" and "dev-dependency"?